packages/lib/src/components/Dropin/components/DropinComponent.tsx

```diff
@@ -97,15 +99,26 @@ export class DropinComponent extends Component<DropinComponentProps, DropinCompo
     private getOnOrderCancel = () => {
         if (this.props.onOrderCancel) {
             return (data: onOrderCancelData) => {
-                this.props.onOrderCancel(data);
+                const order = sanitizeOrder(data.order);
+                new Promise<{ amount: PaymentAmount }>((resolve, reject) => {
+                    this.props.onOrderCancel({ order }, { resolve, reject });
+                })
+                    .then(({ amount }) => this.props.elementRef.handleAdvanceFlowPaymentMethodsUpdate(null, amount))
+                    .catch(error => {
+                        console.error(error);
+                        this.setStatus(error?.message || 'error');
+                    });
             };
         }
         if (this.props.session) {
             return (data: onOrderCancelData) =>
                 this.props.session
                     .cancelOrder(data)
                     .then(() => this.props.core.update({ order: null }))
-                    .catch(error => this.setStatus(error?.message || 'error'));
+                    .catch(error => {
+                        console.error(error);
```

Comment:
Is it intentional to leave `console.error` ?

Reply:
Change this to onError and NETWORK_ERROR.